src/tutorial/runtime/summary.rst

```diff
@@ -0,0 +1,38 @@
+Tutorial Summary
+================
+
+You've made it to the end of the Cylc tutorial!
+
+The Cylc tutorial aims to introduce you to the main concepts in Cylc to
+prepare you for working with and writing Cylc workflows.
+But there is so much more we didn't have time for.
+
+:ref:`Rose Tutorial <Rose Tutorial>`_
+   `Rose`_ is a tool for creating configurable applications. It's often used
+   with Cylc to configure more complex tasks, or even the workflow itself.
+   
+   Rose configurations can have metadata, support validation and can be edited
+   using a GUI.
+   
+   You might want to try the :ref:`Rose Tutorial` if these things are of interest.
+:ref:`Further Topics <tutorials.furthertopics>`
+   There some extra tutorials which cover some of the things the main tutorial
+   doesn't in the :ref:`further topics section <tutorials.furthertopics>`.
+`Discourse`_
+   If you get stuck, encounter an issue, have a question, or just fancy a chat
+   about Cylc, feel free to reach out to the Cylc community on our `Discourse`_
+   forum.
+   
+   We also make announcements (e.g. new Cylc releases) on the forum, so it's a
+   good to keep an eye on it.
+:ref:`User Guide <user guide>`
+   There is a comprehensive :ref:`user guide` which goes into Cylc's
+   capabilities in detail.
+:ref:`Workflow Design Guide <workflow design guide>`
+   This covers recommended code style and best practice.
+:ref:`Reference <reference>`
+   The reference section contains all sorts of technical details.
+
+   It also lists all of the configurations Cylc supports for workflow
+   definition :cylc:conf:`flow.cylc`, and for site/user setup
+   :cylc:conf:`global.cylc`.
```

Comment:
Is it worth putting a pointer to the 728 section here too, given that at least a few people on the course have been updaters?